tests/coreos-install/util/util.go

```diff
@@ -165,6 +165,45 @@ func DownloadFile(t *testing.T, tmpDir, name string) error {
 	return nil
 }
 
+// used to create files in /run/systemd/network on the host machine if no
+// files are present for use in network unit testing
+func CreateNetworkUnits(t *testing.T) string {
+	var ret string
+	if _, err := os.Stat("/run/systemd/network"); os.IsNotExist(err) {
+		err = os.MkdirAll("/run/systemd/network", 0777)
+		if err != nil {
+			t.Fatalf("creating /run/systemd/network: %v", err)
+		}
+		ret = "/run/systemd/network"
+	} else {
+		files, err := ioutil.ReadDir("/run/systemd/network")
+		if err == nil {
+			for _, file := range files {
+				if _, err = os.Stat(file.Name()); !os.IsNotExist(err) {
+					// a network unit already exists
+					return ""
+				}
+			}
+		}
+	}
+
+	file, err := os.Create("/run/systemd/network/coreos-install-test")
+	if err != nil {
+		t.Fatalf("creating network unit: %v", err)
+	}
+	defer file.Close()
+
+	_, err = file.WriteString("coreos-install test file")
```

Comment:
If the host is actually using networkd, this has the potential to create confusion.  Best to write a valid file.  This `.network` file should be a no-op:

```
# Created by coreos-install tests
[Match]
Architecture=coreos-install
```